modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/cell_renderers/ImageRenderer.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClaySticker from '@clayui/sticker';
+import React from 'react';
+
+import imagePropsTransformer from '../views/utils/imagePropsTransformer';
+
+function ImageRenderer(sourceImageProps: any) {
+	const imageProps = imagePropsTransformer(sourceImageProps);
+
```

Comment:
Extracted all previous logic to the `imagePropsTransformer` utility.
It only remains `shape` and `size` properties as their are not HTML image attributes. Based on https://clayui.com/docs/components/sticker/api.html, defaults does not seems to be valid.

Reply:
Nice catch with invalid defaults! Can we change L16 default to `'circle'`? We can also remove `rounded` from `interface IImageProps` .

Reply:
We could remove `rounded` but not changing the default one

### rounded in commerce
![rounded](https://github.com/liferay-frontend/liferay-portal/assets/132653342/1a3cbb9d-c29a-419f-8556-66642afe3dfd)

### circle in commerce
![circle](https://github.com/liferay-frontend/liferay-portal/assets/132653342/3a4b39c5-cec8-4cbe-98c4-1142c20da8a4)

Reply:
Is `rounded` is the same as undefined or null?

Reply:
It is used to compose a className `sticker-rounded` that actually does nothing. Removing the default just prevent from generating this extra class.